### PR TITLE
Edit comments

### PR DESCRIPTION
**Pull request by Sjmarf**
See also [the accompanying MlemMiddleware PR](https://github.com/mlemgroup/MlemMiddleware/pull/48).

### CODE REVIEW

File: Mlem.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -2396,8 +2412,8 @@
 			isa = XCRemoteSwiftPackageReference;
 			repositoryURL = "https://github.com/mlemgroup/MlemMiddleware.git";
 			requirement = {
-				kind = upToNextMinorVersion;
-				minimumVersion = 0.27.0;
+				branch = sjmarf/edit;
+				kind = branch;
```
Comment (EricBAndrews) [inline Mlem.xcodeproj/project.pbxproj]: Reminder

**Review by EricBAndrews — CHANGES_REQUESTED** [review]
Looks good!

Editing only appears to work in profile, however--comments under a post are only `Comment1`, so cannot open the sheet (despite presenting the option):


https://github.com/user-attachments/assets/ada0c214-4801-47aa-9fbf-38d22a93e4ff

**Comment by Sjmarf** [comment]
Fixed! Relies on [this MlemMiddleware PR](https://github.com/mlemgroup/MlemMiddleware/pull/49)

**Review by EricBAndrews — APPROVED** [review]
LGTM! 🔥

**Pull request merged**